tools/plugins/com.liferay.ide.service.core/META-INF/MANIFEST.MF

```diff
@@ -6,6 +6,7 @@ Bundle-Version: 3.6.1.qualifier
 Bundle-Activator: com.liferay.ide.service.core.ServiceCore
 Bundle-Vendor: %bundle.vendor
 Require-Bundle: com.liferay.ide.core,
+ com.liferay.ide.gradle.core,
```

Comment:
@Seiphon  @jtydhr88   We need to refactor this so that the service.core doesn't need to depend on gradle.core . Perhaps a new interface or service needs to be added.